packages/verkle/README.md

```diff
@@ -80,6 +80,50 @@ const { EthereumJSClass } = require('@ethereumjs/[PACKAGE_NAME]')
 
 Using ESM will give you additional advantages over CJS beyond browser usage like static code analysis / Tree Shaking, which CJS cannot provide.
 
+## Debugging
+
+The `Verkle` class features optional debug logging.. Individual debug selections can be activated on the CL with `DEBUG=ethjs,[Logger Selection]`.
+
+`ethjs` **must** be included in the `DEBUG` environment variables to enable **any** logs.
+Additional log selections can be added with a comma separated list (no spaces). Logs with extensions can be enabled with a colon `:`, and `*` can be used to include all extensions.
+
+`DEBUG=ethjs,thislog,thatlog,otherlog,otherlog:sublog,anotherLog:* node myscript.js`
+
+The following options are available:
+
+| Logger              | Description                                      |
+| ------------------- | ------------------------------------------------ |
+| `verkle`            | minimal info logging for all verkle methods      |
+| `verkle:<METHOD>`   | debug logging for specific verkle method         |
+| `verkle:<METHOD>:*` | verbose debug logging for specific verkle method |
+| `verkle:*`          | verbose debug logging for all verkle methods     |
+
+To observe the logging in action at different levels:
+
+Run with minimal logging:
+
+```shell
+DEBUG=ethjs,verkle npx vitest test/verkle.spec.ts
+```
+
+Run with **put** method logging:
+
+```shell
+DEBUG=ethjs,verkle:PUT npx vitest test/verkle.spec.ts
```

Comment:
Ah, ok, just wrote the lowercase argument, now seeing this here: I am pretty sure though, that we already had this as common ground, just some inconsistent introduction here. Can you also please just do `PUT` -> `put` and the like here along this work?

Reply:
I went ahead and reverted changing the examples to use lowercase for the method names, but I made the namespaces we have control over all uniformly use lowercase letters.

Reply:
Ok, nice 🙂